schema.sql

```diff
@@ -31,4 +31,30 @@ ALTER TABLE animals DROP COLUMN species;
 
 ALTER TABLE animals ADD species_id integer, ADD FOREIGN KEY (species_id) REFERENCES species(id);
 
-ALTER TABLE animals ADD owner_id integer, ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
\ No newline at end of file
+ALTER TABLE animals ADD owner_id integer, ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
+
+-- Day4
+
+CREATE TABLE vets (
+    id integer PRIMARY KEY,
+    name VARCHAR(150) NOT NULL,
+    age integer NOT NULL,
+    date_of_graduation date NOT NULL
+);
+
+CREATE TABLE specializations (
+    id integer PRIMARY KEY,
+    species_id integer NOT NULL,
+    vet_id integer NOT NULL,
+    FOREIGN KEY (species_id) REFERENCES species (id),
+    FOREIGN KEY (vet_id) REFERENCES vets (id)
+);
+
+CREATE TABLE visits (
+    id integer PRIMARY KEY,
```

Comment:
- [ ] As I mentioned in previous comments, this id should be autoincremented PRIMARY KEY.